Wokoro/way_farer_test
# #167307170 User booking deleting

#### What does this PR do?
PR enables signed-in users to delete their bookings, through `/api/v1/bookings/:booking_id`, end-point.

#### Description of Task to be completed?
A valid user of the way_farer platform should be able to delete his/her booking through a dedicated endpoint, in this case `/api/v1/bookings/:booking_id`.

#### How should this be manually tested?
Using postman, the endpoint can be tested using the following steps
* Sign in to the application with a valid user email and password, can create one if the user does not exist.

* Create a booking to be deleted if none exist. Booking id should be noted!

* Navigate to  `/api/v1/bookings/:booking_id`, input the booking id as a path parameter, and send.

#### Any background context you want to provide?
A restriction is placed on users to allow them to delete only booking made by them.

#### What are the relevant pivotal tracker stories?
[#167307170](https://www.pivotaltracker.com/story/show/167307170)
[#167307176](https://www.pivotaltracker.com/story/show/167307176)

#### Screenshots (if appropriate)
none

#### Questions:
none
